Library/Formula/websockify.rb

```diff
@@ -0,0 +1,71 @@
+class Websockify < Formula
+  homepage "https://github.com/kanaka/websockify"
+  url "https://github.com/kanaka/websockify/archive/v0.6.0.tar.gz"
+  sha1 "f2bffebc9e4d94ad3a329638bee63e899eaad9c0"
+
+  depends_on :python if MacOS.version <= :snow_leopard
+  depends_on "openssl"
+
+  def install
+    system "make"
+
+    Dir.chdir("other") do
+      system "make"
+    end
+
+    ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python2.7/site-packages"
+    system "python", *Language::Python.setup_install_args(libexec)
+    bin.install Dir["#{libexec}/bin/*"]
+    bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
+
+    # mv this so it is not installed with the other docs
+    mv "docs/websockify.1", "websockify.1"
+    man1.install "websockify.1"
+    (doc+"websockify").install "docs"
+
+    # Upstream Bug: Not named ".dylib"
+    lib.install "rebind.so"
+
+    # Install C version of websockify
+    mv "other/websockify", "other/websockify-c"
+    bin.install "other/websockify-c"
+
+    # Clean up before installing client
+    mkdir "client"
+    mv "wsirc.html", "client/wsirc.html"
+    mv "wstelnet.html", "client/wstelnet.html"
+    # rmdir, not rm_r, as this is genuinely empty and we want to fail if the tarball
+    # ever changes in a new release
+    rmdir "include/web-socket-js-project"
+    mv "include", "client/include"
+    (share+"websockify").install "client"
+
+    # Clean up before installing node js
+    mv "other/js", "node-js"
+    (share+"websockify").install "node-js"
+
+    # Clean up before installing Ruby Websockify
+    mkdir "ruby"
+    mv "other/websocket.rb", "ruby/websocket.rb"
+    mv "other/websockify.rb", "ruby/websockify.rb"
+    (share+"websockify").install "ruby"
+
+    # Clean up before installing Clojure Websockify
+    mkdir "clojure"
+    mv "other/project.clj", "clojure/project.clj"
+    mv "other/websockify.clj", "clojure/websockify.clj"
+    (share+"websockify").install "clojure"
+
+    # Binaries not installed in bin (as bin.install ):-
+    # - rebind: coding of discovery of rebind.so needs re-writing
+    # - other/wswrap: dependency wswrapper.so seems to be missing
+    # - other/launch.sh: coding of path discovery of dependencies is just too weird to work
+    (share+"websockify"+"scripts").install "rebind"
+    (share+"websockify"+"scripts").install "other/wswrap"
+    (share+"websockify"+"scripts").install "other/launch.sh"
+  end
+
+  test do
+    system "websockify", "-h"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!